site/content/docs/api-checks/api-checks.md

```diff
@@ -8,16 +8,67 @@ menu:
     identifier: overview-api-checks
 ---
 
-API checks consist of two parts:
+API checks consist of few parts:
 
-**1. An HTTP request** 
+> The following phases are executed under the hood by the Node.js [request](https://www.npmjs.com/package/request/v/2.88.2) library and the [chai.js](https://www.npmjs.com/package/chai/v/4.3.4) assertions library. All public checks are executed on Amazon's AWS Lambda infrastructure.
```

Comment:
Should we point out [request](https://www.npmjs.com/package/request)? As it's deprecated for a while I'm not sure if we want to point this out. :)

Reply:
@stefanjudis let's be transparent here - it's true that request is deprecated but it is still widely used and understood.